packages/react-devtools-shared/src/__tests__/profilingUtils-test.js

```diff
@@ -16,7 +16,7 @@ describe('profiling utils', () => {
 
   it('should throw if importing older/unsupported data', () => {
     expect(() =>
-      utils.prepareProfilingDataFrontendFromExport(
+      utils.prepareProfilingDataFrontendFromImport(
```

Comment:
This was more semantically correct before.

We're *importing* data, but we're reading *from an export*.

If you want to rename it because you find the name confusing, you could name it `prepareProfilingDataFrontendForImport` I guess.